proxies/live/apiproxy/targets/target.xml

```diff
@@ -45,11 +77,6 @@
     </HTTPTargetConnection>
   -->
   <HTTPTargetConnection>
-    <URL>http://mocktarget.apigee.net</URL>
-    <Properties>
-      <Property name="supports.http10">true</Property>
-      <Property name="request.retain.headers">User-Agent,Referer,Accept-Language</Property>
-      <Property name="retain.queryparams">apikey</Property>
-    </Properties>
+    <URL>https://comms-apim.de-apim.communications.national.nhs.uk</URL>
```

Comment:
We can reference the target server by it's name listed in the infra repo, see my PR [here](proxies/live/apiproxy/targets/target.xml)